app/assets/javascripts/services/topology_service.js

```diff
@@ -65,5 +65,29 @@ ManageIQ.angular.app.service('topologyService', function() {
     return { x: 0, y: 9, r: 17 };
   };
 
+  this.getItemStatusClass = function(d) {
+    switch (d.item.status) {
+      case "OK":
+      case "On":
+      case "Ready":
+      case "Running":
+      case "Succeeded":
+      case "Valid":
+        return "Success";
```

Comment:
I can make the lowercase change in another patch as it requires backend changes as well, and it was not the focus of this patch.

as for the class definitions, not sure what you mean that they are missing, but the stroke colors were defined in css, and remained there, just moved this particular part to a shareable code.